test/integration_test/applicationbackup_test.go

```diff
@@ -360,11 +362,28 @@ func getBackupFromListWithAnnotations(backupList *storkv1.ApplicationBackupList,
 }
 
 func applicationBackupRestoreTest(t *testing.T) {
+	var (
```

Comment:
We can move this to common.go so that other tests like snapshot can use these variables also, in case we decide to do IO in those tests

Reply:
Well, yes I can. I just have a concern if these test run concurrently for data races.

Reply:
These test run sequentially so we should be ok.

Reply:
done